server/services/stadiums.ts

```diff
@@ -109,3 +109,31 @@ export const getStadiumProfileService = async (
     data: stadium,
   };
 };
+
+export const getStadiumsService = async (): Promise<{
+  status: number;
+  data: object;
+}> => {
+  const res = await User.findAll({
```

Comment:
res is not ideal because it does not provide meaningful information about the purpose or content of the variable. A more appropriate name for this variable could be **userStadiums** or something similar.